t/01-pdk/04-request/00-phase_checks.t

```diff
@@ -4,6 +4,7 @@ use Test::Nginx::Socket::Lua;
 use t::Util;
 
 $ENV{TEST_NGINX_HTML_DIR} ||= html_dir();
```

Comment:
This variable becomes unused; this patch should be getting rid of it.

Reply:
Good catch, I didn't check on those. Will update.